bioverse/analysis.py

```diff
@@ -14,10 +17,31 @@
 mp.set_start_method('spawn', force=True)
 
 def test_hypothesis_grid(h, generator, survey, N=10, processes=1, do_bar=True, bins=15, return_chains=False,
-                         mw_alternative='greater', method='dynesty', nlive=100, **kwargs):
+                         mw_alternative='greater', method='dynesty', nlive=100, error_dump_filename: str = None,
+                         seed: int = None, **kwargs):
     """ Runs simulated surveys over a grid of survey and astrophysical parameters. Each time, uses the simulated
     data set to fit the hypothesis parameters and computes the model evidence versus the null hypothesis. """
 
+    # set up logging
+    logger = logging.getLogger(__name__)
+
+    # logger.setLevel(logging.DEBUG)
+
+    if error_dump_filename:
+        handler = logging.FileHandler(error_dump_filename, mode="w")
+
+        # handler.setLevel(logging.DEBUG)
+        logger.addHandler(handler)
+    else:
+        handler = None
+
+    if not seed:
```

Comment:
not sure if we should do that. While we need to draw pseudo-random numbers for the iterations, it is key that runs are 1:1 reproducible. 
I think the previous implementation did that if the numpy rng is seeded globally (but I'm not positive).

Reply:
yeah let's use the default seed we're using elsewhere in the code, e.g. `functions.py`. I'm preparing a commit for this.